nesta/packages/nlp_utils/preprocess.py

```diff
@@ -54,7 +54,7 @@ def clean_and_tokenize(text, remove_stops):
     tokens = tokens_re.findall(text)
     _tokens = [t.lower() for t in tokens]
     filtered_tokens = [token.replace('-', '_') for token in _tokens
-                       if len(token) > 2
+                       if (not remove_stops or len(token) > 2)
```

Comment:
Looks good. My guess is that there are only a few stop words (for example, _of_) that are useful in ngrams. Could you remove those from the `stop_words` instead?

Reply:
actually I did a test and almost every stop word appears in a wiktionary n-gram...